types/mini-css-extract-plugin/index.d.ts

```diff
@@ -4,10 +4,12 @@
 //                 Katsuya Hino <https://github.com/dobogo>
 //                 Spencer Miskoviak <https://github.com/skovy>
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
+//                 James Garbutt <https://github.com/43081j>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.7
 
-import { Configuration, Compiler } from 'webpack';
```

Comment:
I would personally keep the named import.

Reply:
i removed this at the time because the compiler complained at me about interop and what not, ill try put it back